src/writing-policies/go/04-valitation.md

```diff
@@ -0,0 +1,351 @@
+# Writing the validation logic
```

Comment:
A new file will appear when the SUMMARY.md file is fixed. Make sure to do further changes on the file with the right name, and remove the old one.